cms/src/components/Header/Header.tsx

```diff
@@ -20,28 +27,56 @@ import {
 } from './Header.styles';
 
 export function Header() {
+  const [showSidebar, setShowSidebar] = useState<boolean>(false);
+  const [showModal, setShowModal] = useState<boolean>(false);
+
+  const windowWidth = useWindowWidth();
+  const mobile = useMobileDetect().isMobile();
+
+  useEffect(() => {
+    if (windowWidth && windowWidth >= 768) {
+      setShowModal(false);
+      setShowSidebar(false);
+    }
+  }, [windowWidth]);
+
+  const closeModal = () => setShowModal(false);
+
+  const toggleSidebar = () => {
+    setShowSidebar(!showSidebar);
+    setShowModal(!showModal);
+  };
+
   return (
-    <Wrapper>
+    <Wrapper isOpen={showModal}>
       <DesktopContent>
         <Logo>
           <Link href="/">Feeda</Link>
         </Logo>
         <PageTitle title="page name...">Сайт притулку для вуличних тварин Murrfecto</PageTitle>
       </DesktopContent>
-      <MobileHeaderWrapper>
-        <MenuWrapper>
-          <MenuBtn onClick={() => console.log('Open menu')}>
-            <MenuIcon />
-          </MenuBtn>
-        </MenuWrapper>
-        <SearchWrapper>
+
+      <MobileHeaderWrapper isOpen={showModal}>
+        {/*Why it doesnt work?*/}
```

Comment:
what?

Reply:
Got rid of mobile variable and it fixed itself